extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/PersistenceUnitMetadata.java

```diff
@@ -12,4 +12,16 @@
 public interface PersistenceUnitMetadata {
 
     Set<String> getEntityClassNames();
+
+    /**
+     * Convenience: load classes provided by {@link #getEntityClassNames()} using the given ClassLoader.
+     */
+    Set<Class<?>> resolveEntityClasses(ClassLoader classLoader);
```

Comment:
I think we should remove this. The proper way to load the classes will always be through the TCCL.